web/templates/pages/members.html

```diff
@@ -20,13 +20,13 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
```

Comment:
i changed the model to
ACTIVITY_LEVEL = (
			('SLEEPYCAT', 'Sleepy Cat'),
			('LAZYCAT', 'Lazy Cat'),
			('ACTIVECAT', 'Active Cat'),
			('MENTORCAT', 'Mentor Cat'),

and the 
<ul class="about_list">
			{% for member in members_list %}
				{% if member.activity_status == 'MENTORCAT' %}
					<li class="mentor-cat"><a href="{% url 'web.views.member_page' user_id=member.id %}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
				{% elif member.activity_status == 'ACTIVECAT' %}

It does not work, no errors just members page does not present anything, any ideas?